src/main/java/wromaciej/moviesrental/movies/service/CartService.java

```diff
@@ -25,4 +25,10 @@ public double calculateRentalRate(Collection<Integer> movieIds) {
 
         return sum;
     }
+
+    public double calculateRentalRateForDays(int movieId, int rentalDays){
```

Comment:
Format code regularly using IntelliJ (ctrl alt L if I remember correctly). between any operator like * and it's parameter there should be a space.